.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: /
+  schedule:
+    interval: daily
```

Comment:
@JAVGan Very nice idea to sue dependabot 
But please use weekly
And IIRC it can even generate group PR instead of separate PR for each repo.